packages/core/src/network/heartbeat.ts

```diff
@@ -25,12 +25,14 @@ export default class Heartbeat {
     const doPing = async (): Promise<void> => {
       await this.networkPeers.ping(toPing.pop(), async (id: PeerId) => {
         log('ping', id.toB58String())
-        try {
-          await this.interaction.interact(id)
+
+        const pingResult = await this.interaction.interact(id)
```

Comment:
I guess we are not doing this in a lot of places, but maybe it would be worth doing something like
```suggestion
        const pingResult = await this.interaction.interact(id)
          .catch(err => log('ping', err))
```